src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -53,6 +53,7 @@
 #include <mathlib/mathlib.h>
 #include <matrix/math.hpp>
 #include <px4_platform_common/px4_work_queue/ScheduledWorkItem.hpp>
+#include <px4_platform_common/module.h>
```

Comment:
@julianoes I don't know what I am doing here. I added this because `PRINT_MODULE_DESCRIPTION` was not defined, but now I'm getting an error:

```
fatal error: 'arraySize' macro redefined [-Wmacro-redefined]
```

Advice appreciated. Not urgent.

Reply:
I pushed a fix

Reply:
Thank you!